### ISSUE TITLE
Only one filter at once in v4

### ISSUE THREAD

**Issue by gaam24** [Deivu/Shoukaku#150]
On v4 when I change filter by setRotation, setEqualizer etc. when other filter was set before this, it was reset and only new one was apply.

This same is when anyone filter was applied and volume was changed. Clear all filters and set volume.

**Comment by Deivu** [comment]
Will check once able to

**Comment by gaam24** [comment]
I think problem is in there
```
public async setTremolo(tremolo?: FreqSettings): Promise<void> {
     await this.node.rest.updatePlayer({
          guildId: this.connection.guildId,
          playerOptions: { filters: { tremolo }}
     });
     this.filters.tremolo = tremolo || null;
}
```

because when I use updatePlayer() with filters from player class all filters working
```
await player.node.rest.updatePlayer({
     guildId: player.connection.guildId,
     playerOptions: { filters: player.filters}
});
```

**Comment by Deivu** [comment]
Fixed in latest master commits in commits `76dca52` onwards
